### DISCUSSION TITLE
[Suggestion] Add option to prioritise picking up ground item(s) over attacking monster

### DISCUSSION THREAD

**Discussion by harry-graham** [runelite/runelite#15990] [Q&A] [answered]
When killing monsters that drop items, it would be nice to have an option (in the Menu Entry Swapper) to prioritise picking up ground items over attacking any monsters when using left-click.

e.g. suppose you are killing chickens, and want to pick up all of the items that are dropped. Each left-click picks up one of the items, unless another chicken walks over the same cell/square, in which case left-clicking attacks the chicken. It would be helpful to have an option to prioritise picking up ground items over attacking the monsters, so that you can left-click the same cell as the items and monsters are on and it will pick up the items first.

**Comment by Nightfirecat** [comment, marked as answer]
This is already possible via the "Shift click Walk here" option under the NPC section of the Menu Entry Swapper plugin. When enabled, you can hold shift to ensure clicking on an item will pick it up rather than attack an NPC walking over that tile.

**Reply by harry-graham** [reply]
Thank you 🙌🏼 sorry for the noise, I'll ask a question rather than start a discussion next time.
